src/_posts/2020-07-18-packaging-with-packer.md

```diff
@@ -0,0 +1,601 @@
+---
+layout: post
```

Comment:
Please review all instances of the word "Packer" and ensure capitalization is consistent throughout the post.